Backend/app/rider-platform/rider-app/Main/src/Domain/Action/Dashboard/InsertPartnerOrgStation.hs

```diff
@@ -0,0 +1,33 @@
+{-# OPTIONS_GHC -Wno-orphans #-}
+{-# OPTIONS_GHC -Wno-unused-imports #-}
+
+module Domain.Action.Dashboard.InsertPartnerOrgStation (postInsertPartnerOrgStationInsertPartnerOrgStation) where
+
+import qualified API.Types.RiderPlatform.Management.InsertPartnerOrgStation
+import Data.OpenApi (ToSchema)
+import qualified Domain.Types.Merchant
+import qualified Domain.Types.PartnerOrgStation as TPOS
+import qualified Environment
+import EulerHS.Prelude hiding (id)
+import Kernel.Types.APISuccess (APISuccess (..))
+import qualified Kernel.Types.Beckn.Context
+import Kernel.Types.Common
+import qualified Kernel.Types.Id
+import Servant
+import qualified Storage.Queries.PartnerOrgStation as QPOS
+import Tools.Auth
+
+postInsertPartnerOrgStationInsertPartnerOrgStation :: (Kernel.Types.Id.ShortId Domain.Types.Merchant.Merchant -> Kernel.Types.Beckn.Context.City -> API.Types.RiderPlatform.Management.InsertPartnerOrgStation.InsertData -> Environment.Flow Kernel.Types.APISuccess.APISuccess)
+postInsertPartnerOrgStationInsertPartnerOrgStation _ _ req = do
```

Comment:
check the `stationId` city and the city from dashboard api are same, 
user may not have accees to create/update partner org stations for some city.